app/models/planet.rb

```diff
@@ -0,0 +1,9 @@
+class Planet < ApplicationRecord
+  has_many :bookings
+  belongs_to :users
```

Comment:
we need to add the relationship to the users through the bookings table as well. The model should contain `has_many :users, through :bookings` as well.